internal/expr/parser_helper_test.go

```diff
@@ -169,3 +175,65 @@ func (s *ExprInternalSuite) TestRemoveComments(c *C) {
 		}
 	}
 }
+
+func (s *ExprInternalSuite) TestParseSliceRange(c *C) {
```

Comment:
I think these tests would really fit better in `expr_test.go` rather than here. The current convention is that the `skip` functions are tested here with white box testing and and the `parse` functions are tested more a black box strategy in `expr_test`. These should really be done with a black box imo since they are public error messages.

Reply:
I think the best thing would be to move the tests with the error messages and keep the other ones here as they are testing that the parsing is the correct one, what do you think?

Reply:
That's fair, I think this layout makes sense.